stable/prometheus/templates/alertmanager-clusterrolebinding.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.alertmanager.enabled -}}
 {{- if .Values.rbac.create }}
```

Comment:
@schoofsc 
Why don't you combine two condition to single 'and'?

Reply:
@zanhsieh  I like it better because it's easier to keep track of. (Maybe only my opinion)